### DISCUSSION TITLE
"Chat Settings" tab missing on Web UI

### DISCUSSION THREAD

**Discussion by Seizure** [oobabooga/text-generation-webui#3150] [Q&A] [answered]
Hello. A friend helped me set this up, but he says that on his computer, he has a "chat settings" tab that lets him set up context, characters, and other things for the AI. But I don't have this tab. Can someone explain what might be going on? Thank you.

**Comment by missionfloyd** [comment, marked as answer]
Edit `webui.py` and add `--chat` on [line 14](https://github.com/oobabooga/one-click-installers/blob/bb79037ebd2d602614a6a19ff28ac4fcc98e1ded/webui.py#L14).
`CMD_FLAGS = '--chat'`

Or go to the Session tab and set mode to chat.

**Reply by Seizure** [reply]
Thanks! That was it. Guess my friend forgot to relay that part.
